recipe/build_base.sh

```diff
@@ -1,22 +1,14 @@
 #!/bin/bash
 
-cat <<EOF > conda_mpl_config.cfg
-[directories]
-basedirlist = $PREFIX
+set -ex
 
-[packages]
-tests = False
-toolkit_tests = False
-sample_data = False
+export AR=$GCC_AR
 
-[libs]
-system_freetype = True
+export MESON_ARGS="${MESON_ARGS} --buildtype=release --prefix=${PREFIX} -Dlibdir=lib --pkg-config-path=${PREFIX}/lib/pkgconfig -Dsystem-freetype=true -Dsystem-qhull=true"
 
-EOF
-
-cat conda_mpl_config.cfg
-sed -i.bak "s|/usr/local|${PREFIX}|" setupext.py
-
-export MPLSETUPCFG=conda_mpl_config.cfg
-
-$PYTHON -m pip install --no-deps --no-build-isolation -vv .
+mkdir builddir
+$PYTHON -m mesonbuild.mesonmain setup builddir $MESON_ARGS
+$PYTHON -m build --wheel \
+        --no-isolation --skip-dependency-check -Cbuilddir=builddir \
+        || (cat builddir/meson-logs/meson-log.txt && exit 1)
```

Comment:
Can you `cat` the log after the first Meson invocation? Otherwise, we get the second run by meson-python, which has a few cached results.

Reply:
Done in commit: https://github.com/conda-forge/matplotlib-feedstock/pull/373/commits/9d8b89be891ec299764dc46651620ff408660d1b